### DISCUSSION TITLE
Rename `TSDataset` methods: `add_columns_from_pandas` -> `add_features_from_pandas`, `update_columns_from_pandas` -> `update_features_from_pandas`

### DISCUSSION THREAD

**Discussion by d-a-bunin** [etna-team/etna#435] [Improvements]
### Description

Current naming is contradicting the method `drop_features. I think that suggested naming is more consistent.

### Thoughts

We could create new methods and deprecate old naming in a major release.

**Comment by d-a-bunin** [comment]
Created issue #567

**Comment by egoriyaa** [comment]
Solved in #593
